routingpy/routers/opentripplanner_v2.py

```diff
@@ -135,13 +105,13 @@ def directions(
         query = f"""
             {{
                 plan(
-                    date: "{ date.strftime("%Y-%m-%d") }"
-                    time: "{ time.strftime("%H:%M:%S") }"
+                    date: "{ date_time.date().strftime("%Y-%m-%d") }"
```

Comment:
this should do the same as before, just omitting the timezone

Reply:
Yes, we must validate this in the unit tests.